tests/nl/media_player_HassSetVolume.yaml

```
@@ -0,0 +1,14 @@
+language: nl
+tests:
+  - sentences:
+      - "zet TV volume naar 90 procent"
+      - "verander het TV volume naar 90"
+      - "Zet het volume op de TV omhoog naar 90%"
+      - "draai het volume omhoog naar 90 op de TV"
+      - "verlaag het volume van de TV naar 90%"
```

Dit zou ook moeten werken toch?

```suggestion
      - "verlaag het volume van de TV naar 90%"
      - "verlaag het volume naar 90%"
```